mycroft/api/__init__.py

```diff
@@ -86,7 +86,7 @@ def get_response(self, response):
                 and not response.url.endswith("auth/token"):
             self.refresh_token()
             return self.send(self.old_params)
-        raise HTTPError(data, response=response)
+        raise RequestException(data, response=response)
```

Comment:
What is the reasoning for raising RequestException instead of HTTPError here? since HTTPError is a subclass of RequestException is shouldn't be strictly necessary right?

In any case I think the pairing and configuration skills need to be updated before merging this since they check for HTTPError.

Reply:
Nice catch. Yeah, it doesn't make much sense to broaden the raise clause; only the except clauses. I've changed it to how it was before.